docs/source/fitting/fitfunctions/MuH.rst

```diff
@@ -0,0 +1,40 @@
+.. _func-MuH:
+
+===
+MuH
+===
+
+.. index:: MuH
+
+Description
+-----------
+
+Fitting function for use by Muon scientists defined by:
+
+.. math:: A(t)=\frac{A_0}{6}e^{-\lambda t}e^{-\frac{(\sigma t)^2}{2}}\left(1+\cos(\omega_Dt)+2\cos(\omega_Dt/2)+2\cos(\frac32\omega_Dt)\right)
+
+:math:`A_0` is the amplitude,
+
+
+:math:`\lambda` is the exponential decay rate,
+
+:math:`\sigma` is the gaussian decay rate,
+
+:math:`\nu` (MHz) is the oscillating frequency,
+
+and :math:`\phi` (rad) is the phase at time :math:`t=0`.
+
+.. figure:: /images/MuH.png
```

Comment:
The thing is I couldn't preview my own code without a proper compiler. So perhaps you could try it out for me?

would something like this work?

.. plot::
    from mantid import mad
    from mantid.simpleapi import FunctionWrapper
    import matplotlib.pyplot as plt
    x  = np.arange(0.1,16,0.1)
    y = FunctionWrapper(MuH, A0 = 0.5, FreqD = 0.5, Lambda = 0.3, Sigma = 0.05)
    plt.plot(x,y)

Reply:
Ah yes I forgot but you're almost there, this should work: 
```
.. plot::
	
   from mantid.simpleapi import FunctionWrapper
   import matplotlib.pyplot as plt
   import numpy as np
   x = np.arange(0.1,16,0.1)
   y = FunctionWrapper("MuH")
   fig, ax=plt.subplots()
   ax.plot(x, y(x))
```

Reply:
Don't I need the parameters ?

Reply:
You can include parameters if you want - if you don't it will take your default values. If you include parameters that line would be:
`y = FunctionWrapper("MuH", A0 = 0.5, FreqD = 0.5, Lambda = 0.3, Sigma = 0.05, Phi = 0.05)`

Reply:
Also I forgot to add it earlier but to add the labels onto the axis so it matches your original image you can put this at the end of the script: 
```
ax.set_xlabel('t($\mu$s)')
ax.set_ylabel('A(t)')
```

Reply:
Now you have added this, you can remove the image as well